gravitee-apim-console-webui/src/management/api/proxy/endpoints/groups/edit/api-proxy-group-edit.component.spec.ts

```diff
@@ -319,25 +376,30 @@ describe('ApiProxyGroupEditComponent', () => {
         httpTestingController.expectOne({ url: `${CONSTANTS_TESTING.env.baseURL}/apis/${api.id}`, method: 'PUT' });
       });
 
-      it('should disable select', async () => {
+      it('should disable service discovery', async () => {
         await loader.getHarness(MatSlideToggleHarness.with({ selector: '[formControlName="enabled"]' })).then((slide) => slide.toggle());
 
-        expect(
-          await loader
-            .getHarness(MatSelectHarness.with({ selector: '[aria-label="Service discovery type"]' }))
-            .then((select) => select.isDisabled()),
-        ).toEqual(true);
-      });
+        expect((await loader.getAllHarnesses(MatSelectHarness.with({ selector: '[aria-label="Service discovery type"]' }))).length).toEqual(
+          0,
+        );
 
-      it('should mark the form as invalid when the service discovery is enabled but the url is not set', async () => {
```

Comment:
This test was passing but the "real" behavior does not ensure that form will be marked as invalid when selecting service discovery provider.
See https://watch.screencastify.com/v/JYaClfuCZkYKTmjjKPSR

This should be fixed, but I did not want to spend more time fixing an issue not reported by user

Reply:
I edited the link